docs/_templates/layout.html

```diff
@@ -0,0 +1,23 @@
+{% extends "!layout.html" %}
+
+{# Custom CSS overrides #}
+{# set bootswatch_css_custom = ['_static/my-styles.css'] #}
+
+{# Add github banner (from: https://github.com/blog/273-github-ribbons). #}
+{% block header %}
+  {{ super() }}
+  <a href="https://github.com/automl/auto-sklearn"
```

Comment:
should be https://github.com/automl/Auto-PyTorch

Reply:
I totally forgot to change the templates, thanks I will do so.